.gitignore

```diff
@@ -11,12 +11,14 @@ npm-debug.log
 node_modules
 v1/node_modules/*
 v1/npm-debug.log
-
+.github
```

Comment:
The GitHub directory contains CI definitions and should not be ignored (at least not fully).